mautic/mautic
# Prevent date range change by mousewheel or touch

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
A common annoyance, especially to those with a Mac, is that touching the mousewheel or touch pad while over a date-range causes the date to spin wildly months into the past or future. This has resulted in some accidentally slow page loads as months are crunched for charts when not needed.

![craziness](https://i.imgur.com/i6Fu3Rl.gif)

This has been resolved (supposedly) by https://github.com/xdan/datetimepicker/issues/179. But upgrading the library hasn't done it. *sigh*

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Hover over a date field on the dashboard.
2. Flick your mouse wheel (or magic mouse) gently.
3. Days/weeks/months flash by.

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Hover over a date field on the dashboard.
3. Flick your mouse wheel (or magic mouse) gently.
4. Nothing should happen.
5. The date pickers should still function as one would otherwise expect.
